output/secrethub.i

```diff
@@ -4,14 +4,18 @@
 %}
 %include exception.i
 #include "secrethub.h"
+#include <stdio.h>
 
-%exception {
-    clearErr();
-    $action
-    char *errMsg = checkErr();
-    if(strlen(errMsg)) {
-        SWIG_exception(SWIG_RuntimeError, errMsg);
-    }
+%typemap(in, numinputs=0) char **errMessage (char *temp="") {
```

Comment:
Also, I found the notion of an `arginit` typemap in the documentation. Could this be used here, maybe?

Reply:
I'll look into it.

Reply:
It seems that `arginit` can configure the default values for the arguments before they are processed (by other typemaps, I presume?). This however is not necessary in our case.